packages/tables/frontend/sirius-components-tables/src/graphql/mutation/useTableMutation.ts

```diff
@@ -68,8 +71,27 @@ export const useTableMutations = (
     mutationChangeColumnVisibility({ variables: { input } });
   };
 
+  const [mutationResizeRow, mutationResizeRowResult] = useMutation<GQLResizeRowData, GQLResizeRowVariables>(
```

Comment:
Please don't add more coupling between unrelated concepts by adding code in this file. Create your own hook as we always do.